network-api/networkapi/wagtailpages/pagemodels/customblocks/table_block.py

```diff
@@ -0,0 +1,37 @@
+from wagtail import blocks
+from wagtail.contrib.table_block.blocks import TableBlock
+
+
+class WideTableBlock(blocks.StructBlock):
+    """
+    A custom block that contains a table and an option to render it wider than the page body content.
+    """
+
+    table = TableBlock()
+
+    wide = blocks.BooleanBlock(
+        required=False,
+        default=True,
```

Comment:
```suggestion
        default=False,
```

Hi @robdivincenzo!

After taking a look at the [related issue](https://github.com/MozillaFoundation/foundation.mozilla.org/issues/10163) for this PR, it appears that design has requested that the "wide" option be set to `False` as the default. 

Can we please update the default here and also in the custom migration op?  

Thanks!

Reply:
Thanks @danielfmiranda! So what I noticed was that in [the later comments](https://github.com/MozillaFoundation/foundation.mozilla.org/issues/10163#issuecomment-1440244910), the dev/design discussion turned to setting it to `True` by default instead. And I think I agree with that `True` is less disruptive, since the current layout is wide. Do you have any more context on if that changed back after these comments?

Reply:
Thanks @robdivincenzo for linking the comments for additional comments. In that case, leaving it set to `True` makes sense to me! 👍